keps/sig-node/4540-strict-cpu-reservation/README.md

```diff
@@ -298,7 +175,7 @@ No new integration tests for kubelet are planned.
   - CPU Manager works with `strict-cpu-reservation` policy option
 
 - Basic functionality
-1. Enable `CPUManagerPolicyAlphaOptions` feature gate and `strict-cpu-reservation` policy option.
+1. Enable `CPUManagerPolicyBetaOptions` feature gate and `strict-cpu-reservation` policy option.
```

Comment:
no need to call it out; actually the intent of this gate is to allow users to **disable** features which are otherwise enabled by default

Reply:
ok, got you